apps/codeforafrica/src/components/FooterMenu/index.js

```diff
@@ -0,0 +1,52 @@
+import { Link } from "@commons-ui/next";
+import PropTypes from "prop-types";
+import React from "react";
+
+import NavList from "@/codeforafrica/components/NavList";
+import NavListItem from "@/codeforafrica/components/NavListItem";
+
+function FooterMenu({ menu }) {
+  if (!menu?.length) {
+    return null;
+  }
+
+  return (
+    <NavList direction="column">
+      {menu.map((item) => (
+        <NavListItem key={item.label}>
```

Comment:
```suggestion
        <NavListItem sx={{ mb: "20px" }} key={item.label}>
```
In footer each li has `20px` `margin-bottom`. This is different in desktop and mobile navigations & `FooterNavList` is the best place to set it.